tests/mir-opt/inline/dont_ice_on_generic_rust_call.rs

```diff
@@ -8,5 +7,7 @@ use std::marker::Tuple;
 
 // EMIT_MIR dont_ice_on_generic_rust_call.call.Inline.diff
 pub fn call<I: Tuple>(mut mock: Box<dyn FnMut<I, Output = ()>>, input: I) {
+    // CHECK-LABEL: fn call(
+    // CHECK-NOT: (inlined <Box<dyn FnMut<I, Output = ()>> as FnMut<I>>::call_mut)
```

Comment:
```suggestion
    // CHECK-NOT: inlined
```
We should not be too specific in CHECK-NOT, as we may miss a subtle change in the printed string.